aws/sagemaker-code-editor
# patch for ip cve fix

**Description**

- added patch for updating ip@2.0.0 to ip@2.0.1

**Motivation**

There is a CVE for ip@2.0.0 that is fixed with ip@2.0.1

*Issue #, if available:*

*Description of changes:*
created patch for this change, applied patches, then copied all to patched folder

*Testing*
- pulled fresh version of code - oss
- build code `yarn install --pure-lockfile --verbose`
- ran smoke tests to verify prior to adding any patches `yarn smoketest --web --browser chromium`
```
  70 passing (6m)
  8 pending

✨  Done in 514.59s.
```
- cleaned vscode dir
- pull fresh version from source `git submodule update --init`
- applied new patch `quilt push patches/security.diff`
- cd into vscode
- build code `yarn install --pure-lockfile --verbose`
- ran smoke tests with current patch `yarn smoketest --web --browser chromium`

```
  70 passing (6m)
  8 pending

✨  Done in 501.63s.
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
